java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1813,15 +1813,12 @@ public void brpop(BaseClient client) {
 
         // nothing popped out
         assertNull(
-                client
-                        .brpop(new String[] {listKey2}, REDIS_VERSION.isLowerThan("7.0.0") ? 1. : 0.001)
-                        .get());
+                client.brpop(new String[] {listKey2}, REDIS_VERSION.isLowerThan("7.0.0") ? 1. : 0.5).get());
```

Comment:
why?

Reply:
now that we're using the timeout passed in the command, 0.001 or 0.1 is too low, causing us to get into timeouts.

Reply:
But core adds .5 sec to it. It is a real user scenario, GLIDE clients should work fine even with small timeouts like .1 or .0001